packages/admin-app/src/components/AssignmentTable/AssignmentTable.tsx

```diff
@@ -26,24 +33,25 @@ export const AssignmentTable: React.FC<AssignmentTableProps> = ({ data, onChange
     const usersQuery = data();
 
     return (
-        <Box display="flex" minHeight="500px" height="100%" width="100%">
```

Comment:
I hate how the grids do this. This long term might be a component if it crops up in another table.